### PR TITLE
Redis Prometheus dashboard KPI and layout updates

### PR DESCRIPTION
**Pull request by algchoo**
### Description
This PR updates the layout and includes new KPIs for the Redis Prometheus dashboard.

### Changes
* [updated the redis prometheus dashboard screenshot](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/670/commits/32b4e4c0648e68c3aad42f2430657a92da7c58bd)
* [updated layout and kpi metrics for redis prometheus dashboard](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/670/commits/251b0ecdd1fe610adaf25413653cb5e8e4d18ef9)
* [updated the metadata description to include new metrics for the redis prometheus dashboard](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/670/commits/0efae578ee8ec3b3eb929bb827be71d7548acd53) 

### Updated screenshot
redis-prometheus

### Additional KPIs
* Evictions
* Latency Percentiles (Milliseconds)
* Hit Percentage
* Blocked Clients
* Client Recent Max Input Buffer
* Slowlog Length
* Fragmentation Ratio

### CODE REVIEW

File: dashboards/redis/redis-prometheus.json [hunk 1]
```diff
@@ -172,23 +193,329 @@
...
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_blocked_clients",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:235]:
```suggestion
                  "prometheusQuery": "redis_blocked_clients{${Cluster}, ${Location}, ${Namespace}}",
```

File: dashboards/redis/redis-prometheus.json [hunk 2]
```diff
@@ -172,23 +193,329 @@
...
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_keyspace_hits_total / (redis_keyspace_hits_total + redis_keyspace_misses_total)",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:267]:
```suggestion
                  "prometheusQuery": "redis_keyspace_hits_total{${Cluster}, ${Location}, ${Namespace}} / (redis_keyspace_hits_total{${Cluster}, ${Location}, ${Namespace}} + redis_keyspace_misses_total{${Cluster}, ${Location}, ${Namespace}})",
```

File: dashboards/redis/redis-prometheus.json [hunk 3]
```diff
@@ -172,23 +193,329 @@
...
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_slowlog_length",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:298]:
```suggestion
                  "prometheusQuery": "redis_slowlog_length{${Cluster}, ${Location}, ${Namespace}}",
```

File: dashboards/redis/redis-prometheus.json [hunk 4]
```diff
@@ -172,23 +193,329 @@
...
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_latency_percentiles_usec * 0.001",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:326]:
```suggestion
                  "prometheusQuery": "redis_latency_percentiles_usec{${Cluster}, ${Location}, ${Namespace}} * 0.001",
```

File: dashboards/redis/redis-prometheus.json [hunk 5]
```diff
@@ -172,23 +193,329 @@
...
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_evicted_keys_total",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:357]:
```suggestion
                  "prometheusQuery": "redis_evicted_keys_total{${Cluster}, ${Location}, ${Namespace}}",
```

File: dashboards/redis/redis-prometheus.json [hunk 6]
```diff
@@ -172,23 +193,329 @@
...
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_mem_fragmentation_ratio",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:388]:
```suggestion
                  "prometheusQuery": "redis_mem_fragmentation_ratio{${Cluster}, ${Location}, ${Namespace}}",
```

File: dashboards/redis/redis-prometheus.json [hunk 7]
```diff
@@ -172,23 +193,329 @@
...
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "redis_client_recent_max_input_buffer_bytes",
```
Comment (varun-c) [inline dashboards/redis/redis-prometheus.json:420]:
```suggestion
                  "prometheusQuery": "redis_client_recent_max_input_buffer_bytes{${Cluster}, ${Location}, ${Namespace}}",
```

**Review by varun-c — CHANGES_REQUESTED** [review]
Added some suggestions to apply the template variables to the newly added chart widgets similar to how it is applied to the previously existing widgets.

**Review by varun-c — APPROVED** [review]

**Comment by robinreneemac727** [comment]
I’m just go in and block everything and everyone on this site. I absolutely
did not sign up for any of it. I’m not techie

**Pull request merged**